### DISCUSSION TITLE
Sticky notes

### DISCUSSION THREAD

**Discussion by Dr-Sen** [mbrlabs/Lorien#289] [Ideas]
Hey! 

First and foremost, thank you for making this awesome project!
Your project is fascinating, and I would love to use it, but I'm missing one feature.

I'm used to using Miro and Mural who offer a somewhat similar tool kit with the addition of "sticky notes".
Sticky notes are objects with text that the user may drag around. They are amazing for ideation and organisation. I use this function all the time, and a simple brainstorming app doesn't offer the flexibility that these notes offer. 

So if possible, I would love it if you were able to make a sticky note with the following ability:
- it's an object that can be colourised (background and text colour)
- It freely movable (text keeps its position relative to the frame)
- text and note can be scaled

additional options would include:
- text formatting options
- Independent scaling of text and note size
- way to connect notes with arrows (for linking and relinking ideas) (making flow charts and so on)

I hope you consider this feature.
Thank you so much for your hard work and wonderful tool!
If you have any questions, feel free to ask I'll try to answer them as soon as possible :D

**Comment by mbrlabs** [comment]
Thank you, glad you like it :)

Sounds a lot like the text box feature described in #63, which is definitely on my feature wishlist as well. 

What makes these sticky notes sticky though? I mean how are they sticky in relation to what? Do they just always stay in the same corner of the window?

**Reply by Dr-Sen** [reply]
Hey!

Nice to hear from you.
I had a look at the text box feature and that would be a nice start.
However, where I think this idea to be different is in the way it looks and manages.

In essence, text frames are the basses of a sticky note. They are both text frames with the difference that a sticky note aims to fit the text to its box size (by scaling the text) and is primarily handled as a movable object with a shape, fill colour and border colour. Only by double tapping it, you enter the text mode. Also, its frame can be linked to other frames with arrows or so.

It's really the digital version of an IRL sticky note. Hope that this makes sense.
![Screenshot 2024-09-04 093221](https://github.com/user-attachments/assets/14e59e2b-989d-4f87-b71a-cd2604e19e8e)
An example of how I use this function in another program.
